cpp/src/pca/pca_mg.cu

```diff
@@ -128,9 +128,6 @@ void fit_impl(raft::handle_t& handle,
              streams,
              n_streams,
              verbose);
-    for (std::uint32_t i = 0; i < n_streams; i++) {
-      handle.sync_stream(streams[i]);
-    }
```

Comment:
This sync point was unneeded, it was already handled after exiting this branch below.